tests/cli_terraform_test.go

```
@@ -172,3 +173,56 @@
+	if err != nil {
+		t.Fatalf("Failed to get the current working directory: %v", err)
+	}
+
+	// Initialize PathManager and update PATH
+	pathManager := NewPathManager()
+	pathManager.Prepend("../build", "..")
+	err = pathManager.Apply()
+	if err != nil {
+		t.Fatalf("Failed to apply updated PATH: %v", err)
+	}
+	fmt.Printf("Updated PATH: %s\n", pathManager.GetPath())
```

golangci-lint (1.62.2)

use of `fmt.Printf` forbidden by pattern `^(fmt\.Print(|f|ln)|print|println)$`

(forbidigo)